api/main.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/quintilesims/layer0/api/provider/aws"
 	awsclient "github.com/quintilesims/layer0/common/aws"
 	"github.com/quintilesims/layer0/common/config"
+	"github.com/quintilesims/layer0/common/db/tag_store"
```

Comment:
Would it be possible to change the package name to something more concise without the underscore? Even `tags` would be sufficient as it is prefixed by `/common/db/` I would think?

https://golang.org/doc/effective_go.html#package-names